client/state/selectors/are-countries-fetching.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Determines whether the list of countries from the specified type is being fetched.
+ *
+ * @param {Object} state - global state tree
+ * @param {String} type - type of list ('domains, 'payments', or 'sms)
+ * @return {Boolean} true if the list is being fetched, false otherwise
+ */
+export default function areCountriesFetching( state, type ) {
```

Comment:
so is it `are` ( `areCountriesFetching` ) or `is` ( `[ 'countries', type, 'isFetching' ]` )  ? =)
I would just go with `is`...

We have ( just ) other 2 caes like that:
![screen shot 2018-06-07 at 15 51 48](https://user-images.githubusercontent.com/326402/41100733-ce202cda-6a6a-11e8-8ec6-7fb8b50ade20.png)

Reply:
Well semantically speaking it should have been `areCountriesFromTypeFetching()` but it's a bit long. I was just trying to write proper English here by the way 🙂. I'm not sure what's wrong with the other two cases you mentioned.